reporters_db/data/reporters.json

```diff
@@ -2340,6 +2342,7 @@
             "variations": {
                 "Black R.": "Black",
                 "Black Rep.": "Black",
+                "U.S. (Black)": "Black",
                 "U.S.(Black)": "Black"
```

Comment:
I think this should be switched to the U.S. Reporter.

Reply:
In this reporter? https://github.com/freelawproject/reporters-db/blob/1639d20b2747dbd2cd7501af630b54d114037836/reporters_db/data/reporters.json#L21057

Should it be added as an edition? I think this happen with other reporters. e.g. U.S. (Cranch), U.S. (How.), U.S. (Pet.), etc.

Reply:
No.  I think they are all the US reporter,

Reply:
If it says Cranch or Black (US) I think it references the Black reporter -